src/main/java/org/prebid/server/floors/PriceFloorFetcher.java

```diff
@@ -65,14 +68,15 @@ public PriceFloorFetcher(ApplicationSettings applicationSettings,
                              Vertx vertx,
                              TimeoutFactory timeoutFactory,
                              HttpClient httpClient,
-                             JacksonMapper mapper) {
+                             JacksonMapper mapper, PriceFloorTestingProperties testingProperties) {
 
         this.applicationSettings = Objects.requireNonNull(applicationSettings);
         this.metrics = Objects.requireNonNull(metrics);
         this.vertx = Objects.requireNonNull(vertx);
         this.timeoutFactory = Objects.requireNonNull(timeoutFactory);
         this.httpClient = Objects.requireNonNull(httpClient);
         this.mapper = Objects.requireNonNull(mapper);
+        this.testingProperties = testingProperties;
```

Comment:
Probably `Objects.requireNonNull` would be good to avoid NPE.

Reply:
Idea, is that this properties can be null